src/analyses/cfg_dominators.h

```diff
@@ -180,26 +186,22 @@ class dominatorst
   /// Return an iterator node if node is in this dominators set, end() otherwise
   /// Note: O(n), when making many queries against the same set it is probably
   /// worth copying into a std::set
-
   const_iterator find(const T &node) const
   {
     std::less<T> less;
-    // FIXME This works around a bug in other parts of the code
-    // in particular, dependence_graph.cpp,
-    // where iterators to different lists than those that are
-    // stored in this set are passed to find.
-    // The Debug libstdc++ will (correctly!) run into an assertion failure
-    // using std::find. std::less for some reason doesn't trigger this assertion
-    // failure, so we use this as an ugly workaround until that code is fixed.
-
-    // NOLINTNEXTLINE
+    // FIXME This works around a bug in other parts of the code in particular,
+    // dependence_graph.cpp, where iterators to different lists than those that
+    // are stored in this set are passed to find. The Debug libstdc++ will
+    // (correctly!) run into an assertion failure using std::find. std::less for
+    // some reason doesn't trigger this assertion failure, so we use this as an
+    // ugly workaround until that code is fixed.
+	// NOLINTNEXTLINE
```

Comment:
Can you restore the empty line and the indentation here? The empty line was there to separate the explanation from the linter hint.